docs/hello_nextflow/09_hello_nf-core.md

```diff
@@ -244,4 +244,1050 @@ You have a general idea of what nf-core offers and you know how to run an nf-cor
 
 ### What's next?
 
+Celebrate and take another break! Next, we'll show you how to use nf-core tooling to build your own pipeline. 
+
+## Create a basic pipeline from template
+
+The nf-core pipeline template is a standardized framework designed to streamline the development of Nextflow-based bioinformatics pipelines.
+
+Creating a pipeline using the nf-core template is greatly simplified by the nf-core tooling. It will help you create a pipeline using the set framework that can be modified to suit your own purposes.
```

Comment:
Maybe actually introduce the tooling (don't think we did that above). So "nf-core provides some tooling in the form of a command line utility.... yada yada yada".

Reply:
same for template

Reply:
Also say explicitly that we've pre-installed the tools for them